EpsilonCCS/Src/MpptRtr.c

```diff
@@ -0,0 +1,44 @@
+#include "cmsis_os.h"
+
+#include "MpptRtr.h"
+
+void mpptRtrTask(void const* arg)
+{
+    uint32_t prevWakeTime = osKernelSysTick();
+    uint32_t channel = 0;
+
+    for (;;)
+    {
+        osDelayUntil(&prevWakeTime, MPPT_HEARTBEAT_CAN_FREQ);
+        // Allocate CAN Message, deallocated by sender "sendCanTask()"
+        MpptCanMsg* msg = (MpptCanMsg*)osPoolAlloc(canTxPool);
+        // Populate CAN Message
+        msg->StdId = MPPT_STDID;
+        msg->Channel = channel;
+        // Update Channel
+        channel = (channel++) % 3;
```

Comment:
Can you set it back to 0 after 3 instead, prevent overflow.

Reply:
Done. Although wouldn't modulus do that anyways? When channel = 3, channel = 3 % 3 = 0?

Reply:
Ah ya you're right, I misread this. Would've been fine as is.